governance/polls/Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021
+summary: MIP39c2-SP24 adds Core Unit IS-001 - Immunefi Security
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp24-adding-immunefi-security-core-unit-is-001/10810
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00
+end_date: 2021-11-22T16:00:00
+---
+# Ratification Poll for Adding the Immunefi Security Core Unit (IS-001) - November 8, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 8 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Immunefi Security Core Unit - IS-001 being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP24
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+The overall goal of the Immunefi Security CU is to secure the Maker ecosystem through multiple angles by leveraging its existing and growing security community and expertise. This involves dedicated services to the Maker ecosystem, effectively establishing a security partnership with the DAO. The CU will also aim to complement and collaborate with existing and future CUs instead of competing with them, leveraging existing processes, infrastructure, and information, among others, in order to deliver greater value to the Maker ecosystem.
+
+As a pioneer in the DeFi space with billions in funds in the protocol, the Maker ecosystem is an attractive target for blackhat hackers who look to steal funds or disrupt the protocol. With millions lost regularly in the cryptocurrency space, it is likely that the Maker protocol, its key people, and its key infrastructure, are targeted on a regular basis.
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md)
```

Comment:
Link should probably be:
https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP39/MIP39c2-Subproposals/MIP39c2-SP24.md